hera_mc/mc_session.py

```diff
@@ -723,3 +724,178 @@ def get_paper_temps(self, starttime, stoptime=None):
 
         return self._time_filter(PaperTemperatures, 'time', starttime,
                                  stoptime=stoptime)
+
+    def add_ant_metric(self, obsid, ant, pol, metric, val):
+        """
+        Add a new antenna metric to the M&C database.
+
+        Parameters:
+        ------------
+        obsid: long integer
+            observation identification number
+        ant: integer
+            antenna number
```

Comment:
I thought about this too. I ended up thinking that because the user will be entering this info as a result of some data analysis, it makes sense to use the antenna numbers in the data file. Of course we can always back out other info given the config management at the time.